sympy/solvers/ode/single.py

```diff
@@ -1059,22 +1059,20 @@ def _verify(self, fx):
         d, e, k = self.wilds_match()
         y = Dummy('y')
         x = self.ode_problem.sym
-        g = simplify(e/d).subs(fx, y)
-        h = simplify(k/d).subs(fx, y)
-        if y in h.free_symbols or x in g.free_symbols:
+        self.g = simplify(e/d).subs(fx, y)
+        self.h = simplify(k/d).subs(fx, y)
+        if y in self.h.free_symbols or x in self.g.free_symbols:
             return False
         return True
 
-    def _get_general_solution(self, *, simplify_flag: bool = True):
+    def _get_general_solution(self, *, simplify: bool = True):
```

Comment:
What does `simplify: bool=True` mean? And how is it used (whatever the name) in this function? Perhaps it should just be dropped.

Reply:
I guess this was used to simplify the solution, not sure though. @oscarbenjamin can you please explain a little?

Reply:
I don't see the `simplify` flag being used anywhere. Not sure what it means. Maybe it should be dropped. In `dsolve` simplification is controlled elsewhere...

Reply:
How do you read that syntax? what is 'simplify: bool=True'?

Reply:
Oh, static type-check reads as "simplify is a bool and its value is True by default". Let's drop this arg -- there is no simplification being requested or done by any of these functions.

Reply:
It might be needed for some solvers so I suggest to finish the refactoring before deciding about that. These classes and methods are all internal so it should be safe to delete the argument later.